lib/ccl/functions/getDccAdmissionCheckScenarios.js

```diff
@@ -0,0 +1,176 @@
+const scenarios = [
+  { identifier: 'DE', subtitle: true, enabled: true },
+  { identifier: 'BW', subtitle: false, enabled: true },
+  { identifier: 'HH', subtitle: true, enabled: false }
+]
+
+const descriptor = {
+  name: 'getDccAdmissionCheckScenarios',
+  definition: {
+    parameters: [
+      {
+        name: 'os'
+      },
+      {
+        name: 'language'
+      },
+      {
+        name: 'now'
+      }
+    ],
+    logic: [
+      // label text
+      {
+        declare: [
+          'labelText',
+          {
+            init: [
+              'object'
+            ]
+          }
+        ]
+      },
+      {
+        assign: [
+          'labelText',
+          {
+            call: [
+              '__i18n.getTextDescriptor',
+              {
+                key: 'ADMISSION_SCENARIO_LABEL_TEXT'
+              }
+            ]
+          }
+        ]
+      },
+      // scenario selection
+      {
+        declare: [
+          'scenarioSelection',
+          {
+            init: [
+              'object'
+            ]
+          }
+        ]
+      },
+      {
+        assign: [
+          'scenarioSelection.titleText',
+          {
+            call: [
+              '__i18n.getTextDescriptor',
+              {
+                key: 'ADMISSION_SCENARIO_SELECTION_TITLE_TEXT'
+              }
+            ]
+          }
+        ]
+      },
+      {
+        assign: [
+          'scenarioSelection.items',
+          {
+            map: [
+              scenarios,
+              {
+                script: [
+                  {
+                    declare: [
+                      'scenarioSelectionItem',
+                      {
+                        init: [
+                          'object',
+                          'identifier', {
+                            init: [
+                              'literal',
+                              { var: 'it.identifier' }
+                            ]
+                          },
```

Comment:
no need for init here, just `'identifier', { var: 'it.identifier' }` should do

Reply:
Proposed changes are implemented in https://github.com/corona-warn-app/cwa-app-ccl/pull/36/commits/b1c5d7032a735ead55265e051836a16015d33d0a